Assets/_BForBoss/_Core/Scripts/TimerViewBehaviour.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using TMPro;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class TimerViewBehaviour : MonoBehaviour
+    {
+        private TMP_Text _timerLabel;
+        private float _time;
+        private bool _isRunning;
+
+        public void StartTimer()
+        {
+            _isRunning = true;
+        }
+
+        public void StopTimer()
```

Comment:
Do you think `PauseTimer()` would be more accurate

Reply:
i'll update it in next pr